project.yml

```diff
@@ -56,7 +56,7 @@ commands:
   - name: create_kb
     help: "Creates KB utilizing SQLite database with Wiki content."
     script:
-      - "env PYTHONPATH=scripts python ./scripts/create_kb.py ${vars.vectors_model} ${vars.language}"
+      - "env PYTHONPATH=scripts python ./scripts/cli_create_kb.py ${vars.vectors_model} ${vars.language}"
     deps:
       - "output/${vars.language}/wiki.sqlite3"
     outputs:
```

Comment:
Should the Annoy index also be removed when running `delete_db`?

Reply:
I added a `delete_embeddings_index` step. Now we have `delete_db`, `delete_embeddings_index` and `clean` (the last of which removes the `kb` and `nlp` folders). It seems kind of arbitrary what's included in `clean` though, no? Maybe we can come up with a better naming?

Reply:
I don't care strongly either way, to be perfectly honest... I'll go ahead and resolve? I think it's fine.